src/helpers/numberFormat.ts

```diff
@@ -1,6 +1,6 @@
-export function numberFormatter(language: string | undefined, humanize: boolean, round: boolean) {
+export function numberFormatter(language: string | undefined, compact: boolean, round: boolean) {
```

Comment:
Unless people are familiar with the Intl.NumberFormat API they may not know that `compact` corresponds to humanizing the value.

This looks fine though.  I like the flexibility.